packages/react/src/radio/Radio.css.ts

```diff
@@ -0,0 +1,70 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+
+export const optionWrapper = style({
+  alignItems: "center",
+  flexDirection: "row",
+  gap: "8px",
+  selectors: {
+    ["&:not(:first-child)"]: {
+      marginTop: "12px",
+    },
```

Comment:
why do we need to do this? can we use `gap` instead of margin

Reply:
Yes Vai we can. Instead of using box, I used Flex in the updated implementation.